src/Logic/operate.js

```diff
@@ -0,0 +1,30 @@
+const Big = require('big.js');
+
+
+export default function operate(numberOne, numberTwo, operation) {
```

Comment:
Let practice Dry Code -> do not repeat;
you can create a variable that will hold new Big(numberOne) and (new Big(numbeTwo).
Of which if you're using `big-js`, I don't think you need `toFixed` and `new Big`, new Big can just be `Big(numberOne)`

Reply:
when i tried to print out the results it wasnt  letting me do it with Big Object. once i used -toFixed i was on the way i needed to put them out. without it is a matter on a option that i have right now. pls if you know one better you could say that. thanks for Dry code practice btw :D 👍